commands/monitoring/pipeline/get.go

```diff
@@ -0,0 +1,62 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/commands/monitoring/pipeline/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func MonitoringFindByIdCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "monitoring",
+		Resource:  "pipeline",
+		Verb:      "get",
+		Aliases:   []string{"g"},
+		ShortDesc: "Retrieve a pipeline",
+		Example:   "ionosctl monitoring pipeline get --pipeline-id GATEWAYID --location de/txl",
+		PreCmdRun: func(c *core.PreCommandConfig) error {
+			if err := core.CheckRequiredFlags(c.Command, c.NS, constants.FlagPipelineID); err != nil {
+				return fmt.Errorf("missing pipeline ID flag:\n %w", err)
```

Comment:
Here `CheckRequiredFlags` actually builds a readable error based on what flags are not set, so you can just return the error as-is

However: This is indeed not a go-ish behaviour. Your approach to do this error wrap is correct (though I would have dropped the newline), but in this specific instance, with this func, let's just return `err`